packages/core/src/checkout-buttons/checkout-button-options.ts

```diff
@@ -122,13 +122,14 @@ export interface BaseCheckoutButtonInitializeOptions extends CheckoutButtonOptio
      * The options that are required to facilitate PayPal Commerce. They can be omitted
      * unless you need to support Paypal.
      */
+    // TODO: should be removed when PAYPAL-1539 hits Tier3
```

Comment:
Is there a JIRA attached to this todo? Also has the change already hit T3?

Reply:
Yes. Here it is:
https://bigcommercecloud.atlassian.net/browse/PAYPAL-1541